### DISCUSSION TITLE
How to detect when editorjs finishes to load and renders ?

### DISCUSSION THREAD

**Discussion by Aarbel** [codex-team/editor.js#1967] [Q&A]
Hi Codex Team,

I need to detect when editorjs finishes to load to adapt my interface layout.

Is there a way to do it easily with editorjs API, and not vanilla JS ?

Thanks a lot

**Comment by nenad-coniq** [comment]
Hey @Aarbel 

You can do that through `onReady` callback on Editor's configuration object.

Take a look at the [documentation](https://editorjs.io/configuration#editor-ready-callback).

```
var editor = new EditorJS({
   // Other configuration properties

   /**
    * onReady callback
    */
   onReady: () => {
      console.log('Editor.js is ready to work!')
   }
});
```
